helm/daemon/keygen.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
changing this to `#!/usr/bin/env bash` would make this script available for broader audience ;)

Reply:
i.e. mina-foundation buildkite checks for helm-chart-lint-steps